editor/gui/scene_tree_editor.cpp

```diff
@@ -508,6 +486,36 @@ void SceneTreeEditor::_add_nodes(Node *p_node, TreeItem *p_parent) {
 	}
 }
 
+void SceneTreeEditor::_update_node_tooltip(Node *p_node, TreeItem *p_item) {
+	// Display the node name in all tooltips so that long node names can be previewed
+	// without having to rename them.
+	String tooltip = String(p_node->get_name());
+
+	if (p_node == get_scene_node() && p_node->get_scene_inherited_state().is_valid()) {
+		p_item->add_button(0, get_editor_theme_icon(SNAME("InstanceOptions")), BUTTON_SUBSCENE, false, TTR("Open in Editor"));
+		tooltip += String("\n" + TTR("Inherits:") + " " + p_node->get_scene_inherited_state()->get_path());
```

Comment:
```suggestion
		tooltip += String("\n") + TTR("Inherits:") + " " + p_node->get_scene_inherited_state()->get_path();
```
Same generally in the changed code, as we're moving it we can improve the readability

Reply:
Thanks！

Reply:
`vformat()` would be the most readable I think.